Parsedown.php

```diff
@@ -75,6 +75,25 @@ function setUrlsLinked($urlsLinked)
 
     protected $urlsLinked = true;
 
+    function setSafeLinksEnabled($safeLinksEnabled)
+    {
+        $this->safeLinksEnabled = $safeLinksEnabled;
+
+        return $this;
+    }
+
+    protected $safeLinksEnabled = true;
+
+    protected $safeLinksWhitelist = array(
```

Comment:
I.e. you could fix a bug in the filter implementation that wouldn't have to be reflected in the extension if they're just adding data.